src/nunavut/cli.py

```diff
@@ -93,7 +93,8 @@ def _build_post_processor_list_from_args(args: argparse.Namespace) \
         'templates_dir': (pathlib.Path(args.templates) if args.templates is not None else None),
         'trim_blocks': args.trim_blocks,
         'lstrip_blocks': args.lstrip_blocks,
-        'post_processors': _build_post_processor_list_from_args(args)
+        'post_processors': _build_post_processor_list_from_args(args),
+        'additional_globals': {'little_endian_support': args.little_endian_support},
```

Comment:
This was the easiest way to add in the ability to check in the templates that I found. I didn't see a different way.

Reply:
This isn't going to provide a complete solution since it only provides the global in the commandline. It also supposed the language being generated can support little-endianess. It seems I need to add a concept of "language features" to support this any any other language-specific optimizations we might add in the future. Let me take a thwack at this...

Reply:
Please take a look at https://github.com/thirtytwobits/nunavut/commit/0ff7f9b9f08781f4819bd40c04e44bdcfe68727c which is a proposed way to plumb through opaque arguments from a library caller or CLI to a `nunavut.lang.Language` object. If we are okay with this design I'll complete this by adding a "known" argument called "target_arch" that is a map of values one of which (the only for now) will be the endianess. Templates will then be able to access this as `{% if target_arch.endian == 'little' %}` and defaults will be provided in the language properites.ini.

Reply:
See #124